generic/release-checklist.yaml

```diff
@@ -0,0 +1,8 @@
+vars:
+  do_fast_track: true
```

Comment:
This is going to be common to all our templates, so maybe we should move its definition out of the individual YAML configs into `config.yaml`.

Reply:
Will do in a later PR.